packages/nimble-components/src/dialog/tests/dialog-matrix.stories.ts

```diff
@@ -28,6 +44,33 @@ const component = html`
     </${dialogTag}>
 `;
 
+const dialogSizingTestCase = (
+    [widthLabel, widthStyle]: [string, string],
+    [heightLabel, heightStyle]: [string, string],
+    [maxHeightLabel, maxHeightStyle]: [string, string]
+): ViewTemplate => html`
+    <p style="font: var(${cssPropertyFromTokenName(
+        tokenNames.bodyFont
+    )}); padding-bottom: 1000px;">${() => widthLabel}; ${() => heightLabel} ${() => maxHeightLabel}</p>
+    <style>
+        ${dialogTag}::part(control) {
+            ${() => widthStyle};
+            ${() => heightStyle};
+            ${() => maxHeightStyle};
+        }
+    </style>
```

Comment:
can you add a class name to that `p` tag like `spacer` and then move the styles into the style tag, also note the pattern used for the css custom property:

```
.spacer {
   font: var(${bodyFont.cssCustomProperty});
   padding-bottom: 1000px;
}
```